types/google-spreadsheet/google-spreadsheet-tests.d.ts

```diff
@@ -0,0 +1,62 @@
+import { GoogleSpreadsheet } from 'google-spreadsheet';
```

Comment:
Please rename this file to `google-spreadsheet-tests.ts` (drop `.d` from filename, as mentioned in `tsconfig.json` hint